### PR TITLE
Malkov 11-node-frameworks

### PR DESCRIPTION
**Pull request by dzinrai**
In progress

### CODE REVIEW

File: 11-node-frameworks/classworks/Malkov_Aleksei/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  .post('/spent', async (ctx, next) => {
+    let bodyData = ctx.request.body;
+    try {
+      bodyData = await schema.validateAsync(bodyData);
+      console.log(bodyData);
+      let session = ctx.session;
+      session.state = session.state ? [...session.state, bodyData] : [bodyData];
+    }
+    catch (err) {
+      console.log(err);
```
Comment (anstill) [inline 11-node-frameworks/classworks/Malkov_Aleksei/index.js]: Please, send errors to the client as well, to they will be able to understand what went wrong

File: 11-node-frameworks/classworks/Malkov_Aleksei/session-counter.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+let session = require('koa-generic-session');
```
Comment (anstill) [inline 11-node-frameworks/classworks/Malkov_Aleksei/session-counter.js]: Let's compose this and main server realisation into one server. Keep in mind that the session counter should work for every request without adding additional code if you add a new endpoint. TIP: check how body-parser works

**Review by anstill — CHANGES_REQUESTED** [review]
Hey, @dzinrai great wor here. Please check my comments and feel free to ask any questions.

File: 11-node-frameworks/classworks/Malkov_Aleksei/session-counter.js (hunk 2 above)
Comment (dzinrai) [reply 11-node-frameworks/classworks/Malkov_Aleksei/session-counter.js]: i made some changes, maybe i didnt get it, but now it should count on every request on main server

**Review by anstill — APPROVED** [review]
well done 🏄🏻‍♂️

**Pull request merged**